### PR TITLE
CTCTRADERS- 578

### PR DESCRIPTION
**Pull request by kenneth-rayner**
Add new test for unloading rejection

### CODE REVIEW

File: src/test/scala/ctc/pages/Page.scala [hunk 1]
```diff
@@ -104,9 +104,10 @@ trait Page extends Matchers with ScreenShotUtility {
     webDriver.navigate().to(authUrl)
   }
 
-  def authenticate(arrivalId: String) = {
+  def authenticate(arrivalId: String, rejectionJourney: Boolean = false) = {
     goToAuthPage()
-    val redirectUrl = s"${Configuration.settings.applicationsBaseUrl}/$arrivalId"
+    val url         = s"${Configuration.settings.applicationsBaseUrl}/$arrivalId"
+    val redirectUrl = if (rejectionJourney) s"$url/unloading-rejection" else url
```
Comment (PSouthern) [inline src/test/scala/ctc/pages/Page.scala:110]: Could this be refactored into a case statement please?  It'll maintain standards across our other journey test repo's.

File: src/test/resources/features/UnloadingRemarksRejectionWithSeals.feature [hunk 2]
```diff
@@ -0,0 +1,18 @@
+@unloading_remarks
+
+@wip
```
Comment (PSouthern) [inline src/test/resources/features/UnloadingRemarksRejectionWithSeals.feature:3]: @wip tag needs to be removed.

File: run_wip.sh [hunk 3]
```diff
@@ -1,3 +1,3 @@
 #!/usr/bin/env bash
 ./drop_unloading_frontend_data.sh
-sbt -Dbrowser=chrome -Denvironment=local 'testOnly ctc.suites.Runner_WIP'
+sbt -Dbrowser=firefox -Denvironment=local 'testOnly ctc.suites.Runner_WIP'
```
Comment (PSouthern) [inline run_wip.sh:3]: Out of interest, did you have problems running the test on Chrome?  May be worth investigating this as we use Chrome in Jenkins.

Comment (kenneth-rayner) [reply run_wip.sh]: changed back to chrome but for my pc it works on firefox

File: src/test/resources/features/UnloadingRemarksRejectionWithSeals.feature (hunk 2 above)
Comment (kenneth-rayner) [reply src/test/resources/features/UnloadingRemarksRejectionWithSeals.feature]: done

File: src/test/scala/ctc/pages/Page.scala (hunk 1 above)
Comment (PSouthern) [reply src/test/scala/ctc/pages/Page.scala:110]: @kenneth-rayner Here's an example from arrivals journey tests - https://github.com/hmrc/declare-transit-movement-arrival-journey-tests/blob/master/src/test/scala/ctc/pages/Page.scala#L106-L122

I think we should follow the same patterns when possible so it's easier to maintain

**Pull request merged**